tests/by-util/test_mkdir.rs

```diff
@@ -123,3 +125,29 @@ fn test_recursive_reporting() {
         .stdout_contains("created directory 'mkdir_test9/../mkdir_test9_1'")
         .stdout_contains("created directory 'mkdir_test9/../mkdir_test9_1/../mkdir_test9_2'");
 }
+
+#[test]
+fn test_mkdir_trailing_dot() {
+    let scene2 = TestScenario::new("ls");
+    new_ucmd!()
+        .arg("-p")
+        .arg("-v")
+        .arg("mkdir_test10-2")
+        .succeeds();
+
+    new_ucmd!()
+        .arg("-p")
+        .arg("-v")
+        .arg(TEST_DIR10)
+        .succeeds()
+        .stdout_contains("created directory 'mkdir_test10'");
+
+    new_ucmd!()
+        .arg("-p")
+        .arg("-v")
+        .arg(TEST_DIR11)
+        .succeeds()
+        .stdout_contains("created directory 'mkdir_test11'");
+    let result = scene2.cmd("ls").arg("-al").run();
+    println!("ls dest {}", result.stdout_str());
```

Comment:
do you want this print statement in the test case?

Reply:
yeah but I can remove it if you prefer :)